rust/operator-binary/src/hdfs_controller.rs

```diff
@@ -251,7 +255,7 @@ pub async fn reconcile_hdfs(hdfs: Arc<HdfsCluster>, ctx: Arc<Ctx>) -> HdfsOperat
     let resolved_product_image = hdfs
         .spec
         .image
-        .resolve(DOCKER_IMAGE_BASE_NAME, crate::built_info::CARGO_PKG_VERSION);
+        .resolve(DOCKER_IMAGE_BASE_NAME, crate::built_info::PKG_VERSION);
```

Comment:
Where is this coming from?

Reply:
[`built`](https://docs.rs/built/latest/built/#always-available) writes those for us.
We include them with `include!(concat!(env!("OUT_DIR"), "/built.rs"));`